pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3Packages, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  version = "1.0.0";
+  src = fetchurl {
+    url = "https://github.com/egh/ledger-autosync/archive/v${version}.tar.gz";
+    sha256 = "603d768c2b56c0238ee0abc9fb54ffb7cfac7573ffc484db3d94e211e9cf5453";
+  };
+  name = "ledger-autosync";
+  propagatedBuildInputs = with python3Packages ;
+   [ asn1crypto beautifulsoup4 cffi cryptography entrypoints fuzzywuzzy idna jeepney
+     keyring lxml mock nose ofxclient ofxhome ofxparse pbr pycparser secretstorage six ];
+
+  doCheck = false; # Checks require ledger as a python package
+
+  meta = {
+    homepage = https://github.com/egh/ledger-autosync;
+    description = "OFX/CSV autosync for ledger and hledger";
+    license = stdenv.lib.licenses.gpl3;
```

Comment:
Also I think this might actually be `lgpl3` https://github.com/egh/ledger-autosync/blob/6923f26c7cfec5070b3ef16fdf949cba5616a78b/setup.py#L26

Reply:
It's inconsistent. The `LICENSE` file is GPLv3 https://github.com/egh/ledger-autosync/blob/d1b3e069c26fe769971740adc8193bc42ff36c2c/LICENSE
and the `license` field in setup.py is GPLv3, https://github.com/egh/ledger-autosync/blob/master/setup.py#L21
but the classifier field does say LGPLv3 as you noted.

My own opinion is that its safest to mark it as whats in the `LICENSE` file, and anyway that's the more restrictive license.